services/gatekeeper/3.16.4/defaults/cm.yaml

```diff
@@ -25,8 +25,8 @@ data:
       probeWebhook:
         priorityClassName: system-cluster-critical
         image:
-          repository: curlimages/curl
-          tag: 7.88.1
```

Comment:
Would it be possible to use a newer image from docker repository directly - https://hub.docker.com/r/curlimages/curl

Reply:
updated the image to **8.8.0**
`sandhya.ravi@GT9X7CVF5F kommander-applications % trivy image docker.io/curlimages/curl:8.8.0 --vuln-type os --ignore-unfixed | grep Total
2024-07-15T15:47:33+05:30	INFO	Vulnerability scanning is enabled
2024-07-15T15:47:33+05:30	INFO	Secret scanning is enabled
2024-07-15T15:47:33+05:30	INFO	If your scanning is slow, please try '--scanners vuln' to disable secret scanning
2024-07-15T15:47:33+05:30	INFO	Please see also https://aquasecurity.github.io/trivy/v0.53/docs/scanner/secret#recommendation for faster secret detection
2024-07-15T15:47:38+05:30	INFO	Detected OS	family="alpine" version="3.19.1"
2024-07-15T15:47:38+05:30	INFO	[alpine] Detecting vulnerabilities...	os_version="3.19" repository="3.19" pkg_num=32
2024-07-15T15:47:38+05:30	WARN	Using severities from other vendors for some vulnerabilities. Read https://aquasecurity.github.io/trivy/v0.53/docs/scanner/vulnerability#severity-selection for details.
Total: 10 (UNKNOWN: 0, LOW: 0, MEDIUM: 10, HIGH: 0, CRITICAL: 0)`

Reply:
Thank you!